include/libtorrent/alert_types.hpp

```diff
@@ -2378,12 +2378,33 @@ namespace libtorrent
 		int m_peers_idx;
 	};
 
+	struct TORRENT_EXPORT dht_direct_response_alert: alert
```

Comment:
would you mind adding comments immediately above this class definition, documenting it too? (the documentation is generated from these comments, you can use rst).